core/src/display_object/movie_clip.rs

```diff
@@ -2189,6 +2211,26 @@ impl<'gc> MovieClip<'gc> {
         RefMut::map(self.0.write(gc_context), |s| &mut s.drawing)
     }
 
+    /// If `true`, this is clip is being used as one of the four
+    /// `ButtonState`s for an `Avm2Button`. Currently, this flag
+    /// is used to opt out of orphan handling in `avm2::valid_orphan`
```

Comment:
If the above-mentioned flag behavior is correct this documentation should be changed from "is being used as a button state" to "was instantiated by `Avm2Button` from an SWF tag".